source/plugin/basics/items.rst

```diff
@@ -0,0 +1,61 @@
+==================
+Working with Items
+==================
+
+Items are identified through an ``ItemStack``. An ``ItemStack`` is an inventory item with information such as the amount of the item in the stack, the type of the item, and extra data such as durability.
+
+Checking an Item's Type
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Checking the type of the item is very simple. You just need to call the ``getItem()`` method in the ``ItemStack``.
+
+.. code-block:: java
+
+    import org.spongepowered.api.item.ItemType;
+    import org.spongepowered.api.item.ItemTypes;
+    import org.spongepowered.api.item.inventory.ItemStack;
+    
+    public boolean isStick(ItemStack stack) {
+        ItemType type = stack.getItem();
+        return type == ItemTypes.STICK;
+    }
+    
+See how simple that is? Now we know that our item is a stick, but sticks can stack, so how many of them are there?
+
+Getting the Amount of Items
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Getting the amount of items in an ``ItemStack`` is relatively easy as well. The ``getQuantity()`` method provided by ``ItemStack`` will handle this.
+
+.. code-block:: java
+
+    import org.spongepowered.api.item.inventory.ItemStack;
+    
+    public int getAmount(ItemStack stack) {
+        return stack.getQuantity();
+    }
+    
+Modifying Item Data of an ItemStack
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Manipulating data such as durability or the lore of an item is done with a data set. Using the ``getData()`` method, we can get any data type about an item and modify it.
+
+.. code-block:: java
+
+    import org.spongepowered.api.data.manipulator.item.DurabilityData;
+    import org.spongepowered.api.item.inventory.ItemStack;
+    
+    import com.google.common.base.Optional;
+
+    public void setUnbreakable(ItemStack stack) {
+        Optional<DurabilityData> optional = stack.getData(DurabilityData.class);
```

Comment:
If possible, I'd hold off on writing the bit about Data since it's going to be changed for the better by the end of the week (I'm completing implementation for usability this week).